app/ExternalFolder.m

```diff
@@ -0,0 +1,56 @@
+//
+//  ExternalFolder.m
+//  iSH
+//
+//  Created by Noah Peeters on 26.10.19.
+//
+
+#import <Foundation/Foundation.h>
+#include "kernel/fs.h"
+#include "kernel/errno.h"
+#include "fs/path.h"
+
+NSURL* urlForMount(struct mount *mount) {
+    return (__bridge NSURL *) mount->data;
+}
+
+int external_mount(struct mount *mount) {
+    NSURL *url = urlForMount(mount);
+    if ([url startAccessingSecurityScopedResource] == NO) {
+        CFBridgingRelease(mount->data);
+        return errno_map();
```

Comment:
startAccessingSecurityScopedResource doesn't set errno so this will return garbage